spread.yaml

```diff
@@ -32,18 +45,11 @@ suites:
         summary: Integration tests
 
 prepare: |
-    DEBS="git lxd kvm"
-    if ! dpkg -l $DEBS; then
-        apt-get update
-        apt-get install -y $DEBS
-    fi
+    apt update
+    apt install -y git qemu-kvm lxd tree
```

Comment:
The original logic here would prevent the time consuming apt-get update/install every time if the debs were already installed. Was there a reason for the modification?

Reply:
No, I just changed that to get updates. I'll revert that.